src/main/java/com/mentoring/ecommerce/adapter/out/persistence/postgresql/product/ProductPostgreSQLRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.mentoring.ecommerce.adapter.out.persistence.postgresql.product;
+
+import com.mentoring.ecommerce.domain.Product;
+import org.springframework.data.repository.CrudRepository;
+
+public interface ProductPostgreSQLRepository extends CrudRepository<Product, Integer> {
```

Comment:
@Pelicer 
Looking for some `Postgres` implementation like `MongoRepository` I found the generic `CrudRepository`.
Still looking for a more specific one to replace it.

Reply:
Use `JPARepository`: https://docs.spring.io/spring-data/jpa/docs/current/api/org/springframework/data/jpa/repository/JpaRepository.html